### ISSUE TITLE
suggesions on assembly tutorial

### ISSUE THREAD

**Issue by voutcn** [edamame-course/2015-tutorials#1]
Hi all,

By chance I found this series of tutorials and was happy to see that you choose MEGAHIT in the [assembly tutorial](https://github.com/edamame-course/2015-tutorials/blob/master/for_review/josh/2015-06-03-assembling_your_genome.md). Here I have some suggestions for the tutorial.

First the stat in tail of the log file is not for the whole assembled contigs. Many standalone contigs generated in previous iterations will not pass to the subsequent iterations, so the tail of the log only shows the contigs assembled for the last iteration. To calculate assembly stat, I suggest using other tools, say QUAST.

Second, we've just updated MEGAHIT to v0.3-b2, in which the PE info can be used and parameters `-l` is no longer needed. This tutorial uses v0.2.1, so I suggest amending the link in the tutorial to https://github.com/voutcn/megahit/tree/v0.2.1-a.

Cheers,
Dinghua

**Comment by jwsorens** [comment]
Hi Dinghua,

Thank you so much for contacting us! We actually did know about the issue
with the log file and we are using QUAST later to calculate assembly stats.
I've just updated our tutorial to reflect the changes you have mentioned,
including that we are using v0.2.1 and that the new v0.3-b2 can handle PE
information. If you have any further comments we would be happy to hear
them.

Thanks for you help,

Jackson Sorensen and the rest of the EDAMAME team
